rasa/cli/arguments/default_arguments.py

```diff
@@ -27,6 +27,18 @@ def add_model_param(
         )
 
 
+def add_no_plot_param(
+    parser: argparse.ArgumentParser, default: bool = False, required: bool = False,
+):
+    parser.add_argument(
+        "--no-plot",
+        action="store_true",
+        default=default,
+        help=f"Don't render plots of confusion matrix and histogram",
```

Comment:
suggestion: in the future we might plot other things and then the description wouldn't be accurate anymore. We could just keep it general?
```suggestion
        help=f"Don't render plots using matplotlib",
```

Reply:
good suggestion,  maybe I'll write something like "Don't render evaluation plots"